src/components/agenda/Agenda.css

```diff
@@ -55,17 +66,16 @@
   width: 70%;
   height: auto;
   margin: .25rem 0 .25rem 0;
-
+  color: black;
 }
 
 .agenda-item {
     display: flex;
     flex-direction: column;
-    width: 100%;
+    width: 40vw;
```

Comment:
Smart! Should we put a 'max-width' on this for when the app is opened on a larger screen?